README.md

```diff
@@ -11,6 +11,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [JSFoo](https://jsfoo.in/2018/) | 13-14 Sep | Bengaluru | The eighth edition of India’s first JavaScript conference.
 | [PyCon India](https://in.pycon.org/2018/) | 22-26 Aug | Bengaluru International Exhibition Center (BIEC), Bengaluru | The premier conference in India on using and developing the Python programming language. |
 | [Great Indian Developer Summit](http://www.developermarch.com/developersummit/index.html) | 24-27 Apr | Bengaluru | 	With over 40000 attendees benefiting from ten game changing editions, GIDS is the gold standard for India's software professional ecosystem. |
+| [WORLD BLOCKCHAIN TECHNOLOGY CONCLAVE](https://www.goeventz.com/event/the-world-blockchain-technology-conclave-2018/56893) | 22-23 Mar | Bangalore | World Blockchain Technology Conclave aims to connect global blockchain gurus and technology players in this space including emerging startups – with regional business and IT leaders from across key industry verticals. |
```

Comment:
Please remove the all-caps in the conference name.

Reply:
The description, though makes sense, takes 5 lines in markdown. IMO the description should be short enough. Can you reduce the description to 2-3 lines (like other conferences)? Thanks!

Reply:
updated.